src/handlers/sync-config.ts

```diff
@@ -45,13 +37,46 @@ export async function syncConfigs(context: Context) {
   } else {
     const pluralSuffix = prUrls.length > 1 ? "s" : "";
     const prList = prUrls
-      .map((url, i) => {
-        const prNumber = url.split("/").pop();
-        return `${i + 1}. [${prNumber}](${url})`;
+      .map((url) => {
+        return `- ${url}`;
       })
       .join("\n");
-    const message = `✅ Successfully created ${prUrls.length} pull request${pluralSuffix}:\n\n${prList}`;
+    const message = `Successfully created ${prUrls.length} pull request${pluralSuffix}:\n\n${prList}`;
```

Comment:
This is redundant and should be removed. It's clear the pull request is made successfully because the link and icon is right next to it.